CefSharp.BrowserSubprocess.Core/JavascriptObjectWrapper.cpp

```diff
@@ -14,6 +14,22 @@ namespace CefSharp
 {
     void JavascriptObjectWrapper::Bind()
     {
+        auto objectName = StringUtils::ToNative(_object->JavascriptName);
+
+        if (_object->IsNull)
+        {
+            auto javascriptObject = CefV8Value::CreateNull();
+            if (V8Value.get() && !V8Value->HasValue(objectName))
+            {
+                V8Value->SetValue(objectName, javascriptObject, V8_PROPERTY_ATTRIBUTE_NONE);
+            }
+            else
+            {
+                V8Value = javascriptObject;
```

Comment:
I think some of this becomes hard to follow, the approach you've taken might not quite suite the existing model, so should look at refactoring.

Regardless code needs comments.